media/website/content/migration.md

```diff
@@ -8,7 +8,8 @@ weight = 3
 
 ## Apel plugin
 
-The stop times of the latest reported job per site and the time of the latest report to APEL are now stored in a JSON file instead of a SQLite database. Therefore, the config parameter `time_db_path` has to be changed to `time_json_path`.
+- The format of the config file was changed from INI to YAML. An updated example config file can be found in the [Documentation](https://alu-schumacher.github.io/AUDITOR/latest/#apel-plugin).
```

Comment:
Can we use a relative path (`../#apel-plugin`) instead?

Reply:
No idea what you mean.

Reply:
You link to the docs for the apel plugin using this URL: https://alu-schumacher.github.io/AUDITOR/latest/#apel-plugin
This always links to the latest version of the docs, even when users visit the edge or a tagged version. By using a relative URL `[Documentation](../#apel-plugin)` we always link to the same version as the user is currently visiting.

Reply:
Ok, thanks. Done